database/src/migrations/template_methodology_species_validations/picklist_variables/v0.1.ts

```diff
@@ -0,0 +1,327 @@
+  { name: 'Timber' },
+  { name: 'Wetland/Meadow' },
+  { name: 'Willow/Shrub' }
+];
+
+const HabitatSlope = [
+  { name: 'Flat (0%)', description: 'Flat (0%)' },
+  { name: 'Minimum (5-20%)', description: 'Minimum (5-20%)' },
+  { name: 'Moderate (20-50%', description: 'Moderate (20-50%' },
+  { name: 'Steep (>50%)', description: 'Steep (>50%)' }
+];
+
+const ActivityNonTarget = [
+  { name: 'Alert' },
+  { name: 'Avoiding Pests' },
+  { name: 'Basking' },
+  { name: 'Bedding' },
+  { name: 'Building' },
+  { name: 'Caching' },
+  { name: 'Casting' },
+  { name: 'Courting' },
+  { name: 'Denning' },
+  { name: 'Disturbed' },
+  { name: 'Drinking' },
+  { name: 'Excreting' },
+  { name: 'Feeding' },
+  { name: 'Fleeing' },
+  { name: 'Feeding Salmonid' },
+  { name: 'Grooming' },
+  { name: 'Hibernating' },
+  { name: 'Hunting' },
+  { name: 'Ingesting Minerals' },
+  { name: 'Incubating' },
+  { name: 'Living' },
+  { name: 'Migrating Daily' },
+  { name: 'Migrating Seasonally' },
+  { name: 'Reproducing birthing' },
+  { name: 'Reproducing eggs' },
+  { name: 'Rearing' },
+  { name: 'Standing' },
+  { name: 'Security, Habitat' },
+  { name: 'Security and/or Thermal' },
+  { name: 'Thermal, Habitat' },
+  { name: 'Territoriality' },
+  { name: 'Not Traveling' },
+  { name: 'Traveling, Flying' },
+  { name: 'Traveling, Unclassified' },
+  { name: 'Traveling, Walking' },
+  { name: 'Traveling on a Path' },
+  { name: 'Traveling, Running' },
+  { name: 'Traveling, Swimming' },
+  { name: 'Traveling, Heli-Skiing' },
+  { name: 'Traveling, Skiing' },
+  { name: 'Traveling, Snowmobiling' },
+  { name: 'Traveling, Snowshoeing' },
+  { name: 'Traveling, Snow Cat' },
+  { name: 'Urinating' },
+  { name: 'Described in comments' }
+];
+
+const FeatureType = [
+  { name: 'Breeding Site' },
+  { name: 'Maternity Roost' },
+  { name: 'Bat Nursery Roost' },
+  { name: 'Rookery' },
+  { name: 'Courting Site' },
+  { name: 'Feeding Site' },
+  { name: 'Resting Site' },
+  { name: 'Staging Site' },
+  { name: 'Ungulate Winter Range' },
+  { name: 'Hibernaculum' },
+  { name: 'Roost' },
+  { name: 'Wallow' },
+  { name: 'Mineral Lick' },
+  { name: 'Burrow' },
+  { name: 'Den' },
+  { name: 'Lodge' },
+  { name: 'Nest' },
+  { name: 'Nest Tree' },
+  { name: 'Plant Community' },
+  { name: 'Plant Site' },
+  { name: 'Hot Spring' },
+  { name: 'Water' },
+  { name: 'Fisheries Sensitive Feature' },
+  { name: 'Marine Sensitive Feature' },
+  { name: 'Described in Comments' }
+];
+
+const Light = [
+  { name: 'Bright', description: '' },
+  { name: 'Flat', description: '' }
+];
+
+const SignAge = [
+  { name: 'Fresh', description: 'Fresh' },
+  { name: 'Old', description: 'Old' }
+];
+
+const GPSDatum = [{ name: 'NAD83' }, { name: 'NAD27' }, { name: 'WGS84' }];
+
+export const basicNumericValidator = [
+  {
+    column_numeric_validator: {
+      name: '',
+      description: ''
+    }
+  }
+];
+
+export function basicCodeValidator(codeValues: object) {
```

Comment:
Would be good to define the type of the parameter in more detail, so its not just `object`.